docs/a320-simvars.md

```diff
@@ -268,6 +268,39 @@
     - Determines if the FAULT light on the CABIN PRESS MODE SEL pushbutton
       should be on
 
+- A32NX_CABIN_ALTITUDE
```

Comment:
Moved all simvars directly pertaining to the Pressurization system to a separate section at the bottom of the document, as suggested above. I have not, however, added a `PRESS` prefix as I don't personally think it would add any information/clarity.